src/autoupdate.c

```diff
@@ -29,56 +29,76 @@
 
 #include "swupd.h"
 
-static void print_help(const char *name)
+static void print_help(void)
 {
 	print("Usage:\n");
-	print("   swupd %s [options]\n\n", basename((char *)name));
-	print("Help Options:\n");
-	print("   -h, --help              Show help options\n");
+	print("   swupd autoupdate [OPTION...]\n\n");
+
+	global_print_help();
+
+	print("Options:\n");
 	print("       --enable            enable autoupdates\n");
 	print("       --disable           disable autoupdates\n");
 	print("\n");
 }
 
-static int enable;
-static int disable;
+#define FLAG_ENABLE 1
```

Comment:
Start at 2000 to avoid clashing with any ascii code or global flags

Reply:
Okay

Reply:
Done